components/WhyChooseUs/CardItem.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReasonDetails from "./ReasonDetails";
+
+const CardItem = ({ ...item }) => {
+  return (
+    <div className="w-[100%] md:w-80 flex flex-col items-center mt-8 relative animate-enterSlow md:animation-none">
```

Comment:
Consider using the library(react-intersection-observer) from the template section.
The animation/transition can be fired only when the element shows on the screen.